### PR TITLE
Add Script Commands: Spotify Now Playing Song URL to Clipboard #53

### PR DESCRIPTION
**Pull request by jacc**
## Description

Allows the current playing Spotify song to be copied as a URL to the clipboard.

## Type of change

- [x] New script command

## Screenshot

ezgif com-video-to-gif

## Dependencies / Requirements

n/a

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/media/spotify/spotify-now-playing-url.applescript [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/osascript
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Spotify Now Playing Song URL
```
Comment (thomaspaulmann) [inline commands/media/spotify/spotify-now-playing-url.applescript:5]: Let's drop `Spotify` from the command title and add it as package name. It's more aligned with the other `Spotify` commands. Ranking shouldn't be effected.

```suggestion
# @raycast.title Copy Current Playing Song URL
```

File: commands/media/spotify/spotify-now-playing-url.applescript [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.icon images/spotify.png
+
```
Comment (thomaspaulmann) [inline commands/media/spotify/spotify-now-playing-url.applescript:10]:
```suggestion
# @raycast.packageName Spotify
```

**Review by thomaspaulmann — CHANGES_REQUESTED** [review]
Thanks for the updates! One last bit, let's adjust the command title and package name to the other Spotify commands. I forgot to mention this before, my bad!

**Comment by thomaspaulmann** [comment]
Thanks for this wonderful screen recording 🙌

**Comment by jacc** [comment]
Absolutely! Changes added, we should be good to go.

**Review by thomaspaulmann — APPROVED** [review]
Perfect! Thanks for the quick turn-around. Let's get this merged and start sharing some music. I expect some good new songs from you in Slack now ;-)

**Comment by jacc** [comment]
I'll need to join the Slack first and then I'll get to sharing :D

**Pull request merged**